src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -248,9 +248,9 @@
         <summary>SyntaxToken representing the less than token.</summary>
       </PropertyComment>
     </Field>
-    <Field Name="Arguments" Type="SeparatedSyntaxList&lt;ModifiedType&gt;" MinCount="1">
+    <Field Name="Parameters" Type="SeparatedSyntaxList&lt;ParameterSyntax&gt;" MinCount="1">
       <PropertyComment>
-        <summary>SeparatedSyntaxList of ModifiedTypes representing the list of parameters and return type.</summary>
```

Comment:
Was this changed because ModifiedType presented too many barriers to code reuse in practice?

Reply:
Precisely.

---
In reply to: [359004607](https://github.com/dotnet/roslyn/pull/40431#discussion_r359004607) [](ancestors = 359004607)